deploy/charts/monitoring/templates/prometheus-role.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: prometheus-cluster-admin
```

Comment:
Let's add `app.kubernetes.io/name=stackrox` here such it will get cleaned up by the `teardown` workflow script.

Reply:
Done